### PR TITLE
New Feature: Testing for Drag and other Functionalities

### PR DESCRIPTION
**Pull request by German-Cobian**
## What's new
Created a test file tasks_drag.test.js for a file containing the following functions:
- a function for editing the task description.
- a function for updating an item's 'completed' status.
- a function for updating an item's index value upon drag/drop actions.
- the "Clear all completed" function.

## Steps to run the test
From the console run the following commands:
- npm install
- npm run test

## Expected results

image

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [x] There should be one expect per test, kindly ensure this is the case in all test cases.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/app/__test__/tasks_drag.test.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  test('remove all completed elements', () => {
+    const storageWithoutCompleted = remove.completed(ul);
+    expect(storageWithoutCompleted).toEqual(
+      expect.arrayContaining([
+        {
+          description: 'Send Video',
+          completed: false,
+          index: 0,
+        },
+      ]),
```
Comment (mikenath223) [inline src/app/__test__/tasks_drag.test.js:127]: - [ ] There should be one expect per test.

File: src/app/__test__/tasks_drag.test.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    const dropElement = 2;
+
+    const newStorage = drag.swap(dragElement, dropElement, ul);
+
+    expect(newStorage).toEqual(
+      expect.arrayContaining([
+        {
+          description: 'Send Video',
+          completed: false,
+          index: 0,
```
Comment (mikenath223) [inline src/app/__test__/tasks_drag.test.js:99]: - [ ] Kindly fix this also, ensure there is only one expect per test.

File: src/app/__test__/tasks_drag.test.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  test('change the index of the elements on drag', () => {
+    const dragElement = 0;
+    const dropElement = 2;
+
+    const newStorage = drag.swap(dragElement, dropElement, ul);
+
+    expect(newStorage).toEqual(
+      expect.arrayContaining([
+        {
+          description: 'Send Video',
```
Comment (mikenath223) [inline src/app/__test__/tasks_drag.test.js:70]: - [ ] Also, only one expect per test here also.

File: src/app/__test__/tasks_drag.test.js (hunk 1 above)
Comment (miguelesco) [reply src/app/__test__/tasks_drag.test.js:127]: https://jestjs.io/docs/expect#expectarraycontainingarray

**Comment by mikenath223** [comment]
- Note to next reviewer:
Kindly approve as it goes in line with what the jest docs recommends here: https://jestjs.io/docs/expect#expectarraycontainingarray

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**